tinykv/README.md

```diff
@@ -29,17 +28,17 @@ Also, you’d better read the overview design of TiKV and PD to get a general im
 First, please clone the repository with git to get the source code of the project.
 
 ``` bash
-git clone https://github.com/pingcap-incubator/tinykv.git
+git clone https://github.com/vldbss-2021/vldb-2021-labs-beta.git
```

Comment:
Such general description may be moved to root level README?

Reply:
Good idea.